server/server.js

```diff
@@ -19,7 +19,6 @@ const db = require('./db/db.js')
 const FLIC_THRESHOLD_MILLIS = 210 * 1000
 const HEARTBEAT_THRESHOLD_MILLIS = 75 * 1000
 const PING_THRESHOLD_MILLIS = 320 * 1000
-const SESSION_RESET_TIMEOUT = 2 * 60 * 60 * 1000
```

Comment:
I already added this value to the `.env` on the `chatbot-dev` and `chatbot` servers.